### ISSUE TITLE
您好，计算lens_logits 时为什么要多加一个1？

### ISSUE THREAD

**Issue by MTCai** [WingsBrokenAngel/delving-deeper-into-the-decoder-for-video-captioning#6]
lens_logits = tf.nn.softmax(
            -(tf.abs(lens_reshape - self.options.avglen) + 1), -1)

**Comment by WingsBrokenAngel** [comment]
To make softmax smoother which was inspired by [Laplace smoothing](https://en.wikipedia.org/wiki/Additive_smoothing).
